internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestAdminToolEndToEnd.java

```diff
@@ -186,4 +191,43 @@ public void testNodeReplicasReadinessCommand() throws Exception {
             clusterName, "--storage-node", Utils.getHelixNodeIdentifier(Utils.getHostName(), server.getPort()) };
     AdminTool.main(nodeReplicasReadinessArgs);
   }
+
+  @Test(timeOut = 4 * TEST_TIMEOUT)
+  public void testUpdateAdminOperationVersion() throws Exception {
+    Long currentVersion = -1L;
+    Long newVersion = 80L;
+    try (VeniceTwoLayerMultiRegionMultiClusterWrapper venice =
+        ServiceFactory.getVeniceTwoLayerMultiRegionMultiClusterWrapper(
+            new VeniceMultiRegionClusterCreateOptions.Builder().numberOfRegions(1)
+                .numberOfClusters(1)
+                .numberOfParentControllers(1)
+                .numberOfChildControllers(1)
+                .numberOfServers(1)
+                .numberOfRouters(1)
+                .replicationFactor(1)
+                .build());) {
+      String clusterName = venice.getClusterNames()[0];
+
+      // Get the parent con†roller
+      VeniceControllerWrapper controller = venice.getParentControllers().get(0);
+      ControllerClient controllerClient = new ControllerClient(clusterName, controller.getControllerUrl());
+
+      // Setup the original metadata
+      AdminTopicMetadataResponse originalMetadata = controllerClient.getAdminTopicMetadata(Optional.empty());
+      Assert.assertEquals(originalMetadata.getAdminOperationProtocolVersion(), (long) currentVersion);
```

Comment:
Can we do a few things to test backward compatibility:
1. create a new store with ControllerClient, update a random store config like partitionCount.
2. Get the original metadata; I expect the "executionId" and "offset" will become some positive numbers, and "upstreamOffset" in the child controller cluster will become positive.
3. Make an update to change protocol version to 80
4. During verification, after making another read from metadata, we check that the previous 3 metadata doesn't change.

Reply:
Updated, please help check again.

Reply:
LGTM. Thanks.